lib/manager/github-actions/__snapshots__/extract.spec.ts.snap

```diff
@@ -136,3 +136,18 @@ Array [
   },
 ]
 `;
+
+exports[`manager/github-actions/extract extractPackageFile() extracts tag line with double quotes 1`] = `
+Array [
+  Object {
+    "autoReplaceStringTemplate": "{{depName}}@{{#if newDigest}}{{newDigest}}{{#if newValue}} # tag={{newValue}}{{/if}}{{/if}}{{#unless newDigest}}{{newValue}}{{/unless}}",
+    "commitMessageTopic": "{{{depName}}} action",
+    "currentValue": "v0.13.1",
+    "datasource": "github-tags",
+    "depName": "pascalgn/automerge-action",
+    "depType": "action",
+    "replaceString": "pascalgn/automerge-action@v0.13.1\\"",
+    "versioning": "docker",
+  },
+]
+`;
```

Comment:
Is this a merge mistake?

Reply:
hmmm isnt that supposed to be in already on the other task?
https://github.com/renovatebot/renovate/pull/14130